Product_Landing_Page.css

```diff
@@ -0,0 +1,32 @@
+*{
+    margin: 0;
+    position: relative;
+}
+#header-img{
+    width: 450px;
+    height: 80px;
+}
+#video{
+    width: 300px;
+    height: 300px;
+    position: relative;
+    top: 300px;
+}
+
+div {
+    display: flex;
+    flex-wrap: wrap;
+}
+@media (max-width: 600px)
+#header {
+    flex-wrap: wrap;
+}
+header {
```

Comment:
It would be fine and semantically correct if spaces are added between lines 19 and 20, as well as between lines 23 and 24.